meson.build

```diff
@@ -0,0 +1,114 @@
+project('tfs', 'cpp', license: ['gpl-2.0'], default_options: ['cpp_std=c++17'])
+
+fmt_dep = dependency('fmt', version: '>=7.1.0', fallback: ['fmt', 'fmt_dep'])
+pugixml_dep = dependency('pugixml', fallback: ['pugixml', 'pugixml_dep'])
```

Comment:
You can also pass `default_options: ['default_library=static']` anywhere that you are doing subproject fallbacks. That guarantees the subproject is statically linked by default, which is reasonable for something that won't be shared with system software anyway. It also simplifies installation and/or distribution.

Reply:
Done!